src/components/Camera/index.js

```diff
@@ -55,7 +55,7 @@ const CameraPure = ({
   hasGrantedPermission,
   fallbackHeight,
 }) => (
-  <div className={classNames(style.camera, className)}>
+  <div className={classNames(style.camera, style[className])}>
```

Comment:
It could be a bit confused in the future when we want to make some changes here. I suggest changing the prop name here to `variant="docLiveCapture"`, `forLiveCapture=true|false` or anything else to encapsulate the `docLiveCaptureFrame` style